jena-core/src/main/java/org/apache/jena/vocabulary/RDF.java

```diff
@@ -59,6 +69,14 @@ public static Property li( int i )
     public static final Resource    Statement       = Init.Statement();
     public static final Resource    List            = Init.List();
     public static final Resource    nil             = Init.nil();
+    /**
+     * This property is used explicitly in facet restrictions.
+     * Also, it can be used as a literal type
+     * (e.g., {@code 'test'^^rdf:PlainLiteral}) in old ontologies based on RDF-1.0
+     *
+     * @see <a href="https://www.w3.org/TR/rdf-plain-literal">rdf:PlainLiteral: A Datatype for RDF Plain Literals (Second Edition)</a>
+     */
+    public final static Resource    PlainLiteral    = Init.PlainLiteral();
```

Comment:
rdf:PlainLiteral should never appear in RDF data!

RDF plain literals should not be necessary for RDF 1.1 (all literals have a datatype).
RDF 1.1 does not have RDF 1.0 "plain literals" (strings + lang tag strings).

Probably worth adding this to the javadoc.

(I would describe it as "archaic" these days.)

Reply:
`rdf:PlainLiteral` is used by jena-ontapi (https://github.com/apache/jena/blob/main/jena-ontapi/src/main/java/org/apache/jena/ontapi/common/OntVocabulary.java#L361), since it is present in the specification: https://www.w3.org/TR/owl2-quick-reference/#Built-in_Datatypes ( OWL 2 Web Ontology Language Quick Reference Guide )